### PR TITLE
Updated phpunit and mockery

### PR DESCRIPTION
**Pull request by gabriel-caruso**
Updated `phpunit/phpunit` to `~6.0` and `mockery/mockery` to `~1.0`. Also organized `require-dev` section alphabetically.

### CODE REVIEW

File: src/MongolidModel.php [hunk 1]
```diff
@@ -218,7 +218,7 @@ protected function hashAttributes()
      */
     public static function __callStatic($name, $arguments)
     {
-        if ($name === 'shouldReceive') {
```
Comment (gmsantos) [inline src/MongolidModel.php]: @rscafi do you know why StyleCI is enforcing yoda comparison?  We don't use it here

File: composer.json [hunk 2]
```diff
@@ -21,9 +21,9 @@
         "illuminate/queue": "^5.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7",
```
Comment (gmsantos) [inline composer.json]: @gabriel-caruso thanks. We use carret operator here `^`. Also, the dependencies order is defined by importance order.

File: src/MongolidModel.php (hunk 1 above)
Comment (gmsantos) [reply src/MongolidModel.php]: Yes, we do use it https://github.com/leroy-merlin-br/mongolid-laravel/blob/master/.styleci.yml :(

File: composer.json (hunk 2 above)
Comment (gabriel-caruso) [reply composer.json]: Done. Have rollback to importance order, and used carret operator.

File: src/MongolidModel.php (hunk 1 above)
Comment (gabriel-caruso) [reply src/MongolidModel.php]: Do you wan't me to rollback this StyleCI change?

**Comment by gmsantos** [comment]
thank you @gabriel-caruso

**Pull request merged**